tools/ci_connector_ops/ci_connector_ops/pipelines/publish.py

```diff
@@ -0,0 +1,206 @@
+#
```

Comment:
This module holds the 🥩 of the publish logic. I tried to replicate in Python the [`manage.sh` steps](https://github.com/airbytehq/airbyte/blob/cda2b7dde2495a49c95ca331ed11d2348feacb2c/tools/integrations/manage.sh)